samples/subsys/input/input_dump/src/main.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/input/input.h>
+#include <zephyr/sys/printk.h>
+
+static void input_cb(struct input_event *evt)
+{
+	printk("input event: dev=%-16s %3s type=%2x code=%3d value=%d\n",
```

Comment:
Should samples use `printk()`?

Reply:
Instead of log? No particular preference, it's nice for this to have no delay since it responds to an input event, `samples/basic/button` uses `printk`. I can change to LOG and use minimal logging if you think it's better.

Reply:
No, I meant as opposed to using `printf()`.

Reply:
Looks like we have _a ton_ of samples using `printk()` even though it is documented to be used for "Print[ing] kernel message[s]". Please disregard this comment.

Reply:
Ah, sure why not, swapped to `printf` and `stdio.h`.